src/include/iscsi-lio-server/widgets.rb

```diff
@@ -643,6 +643,9 @@ def AddClntDialog
           if @changed_lun.has_key?(s)
             txt = _("Client name already exists!")
           end
+          if s.index /["\*\?"]/
```

Comment:
I am not sure what you want achieve, but constructed regexp looks wrong. Also you do not use index, so I think better and more readable method looks like ( output from irb )
```
irb(main):001:0> "a*b".match /[*?]/
=> #<MatchData "*">
irb(main):002:0> "ab".match /[*?]/
=> nil
irb(main):003:0> "a?b".match /[*?]/
=> #<MatchData "?">

```